Src/Plugins/AzureDevOpsConfiguration/AzureDevOpsConfiguration.csproj

```diff
@@ -11,6 +11,21 @@
     <PackageId>Sarif.PatternMatcher.AzureDevOpsConfiguration</PackageId>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <MSBuildWarningsAsMessages>
+		// MSB3277 - Found conflicts with different versions of the same
```

Comment:
Nit: `.` at the end of the sentence. Also "found conflicts *among* different versions..." might be better.